gubernator/app.yaml

```diff
@@ -1,11 +1,16 @@
-application: k8s-gubernator
-version: test  # production is 1
 runtime: python27
 api_version: 1
 threadsafe: yes
 instance_class: F2
 
+inbound_services:
+- warmup
+
 handlers:
+- url: /_ah/warmup
```

Comment:
I'm just following the warmup example docs: https://cloud.google.com/appengine/docs/standard/python/warmup-requests/configuring

I suspect that `/_ah/warmup` is the URL that appengine calls during migration to warmup your app, and it is up to the app ensure that this calls does what is necessary for warmup.

I expect future PRs to expand this handler to do something more appropriate, but this handler is necessary to make `gcloud app versions migrate` work at all.